### ISSUE TITLE
Suggestion: UI "ADD TO filter option" 

### ISSUE THREAD

**Issue by MarvinFS** [roundcube/roundcubemail#8606]
Dear colleagues,

So, I've been using other mail systems for quite some time and I found, that "add to filter option" is extremely useful in many cases, when you have substantial mailflow. We already have "create filter" button when clicking on message options, but that adds a **NEW** filter to the system. If you are using separate filters for a single email types\addresses soon your filter set will become unmanageable. 
I suggest to implement something like Zimbra has, which conveniently lets you (when you click on message or  address) to **ADD entry to existing filter**. We may re-use existing dialog, where user selects, what exactly he wants to use as filter criteria, and then just open a new entry in existing filter, chosen from the list of existing filters. that would be SO MUCH more convenient, rather then copy data to clipboard, go to settings, open filters, find target filter, edit it add new line, blah blah... you get the point. 

Does it seems reasonable for you? 

Example from Zimbra:
![add_to-filter](https://user-images.githubusercontent.com/7998636/177411563-38bd3c8c-6679-494a-8232-ca1112dd1120.png)

Regards,
MarvinFS

**Comment by alecpl** [comment]
It would make sense when you click on the email address, but what would you expect in the whole message context?

**Comment by MarvinFS** [comment]
> It would make sense when you click on the email address, but what would you expect in the whole message context?

That is correct and subject probably. But these fields are mostly used for filtering and I obviously not saying about spam filtering, as spam filtering by mail address makes no sense, these are more for email workflow convenience, where you separate you flows to folders and categorizes it. If you use folders for different types of messages it helps tremendously.
